open-banking-accelerator/components/event-notifications/com.wso2.openbanking.accelerator.event.notifications.service/src/main/java/com/wso2/openbanking/accelerator/event/notifications/service/dao/PostgreSqlEventSubscriptionDAOImpl.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Shall we change the license header to open source one?

Reply:
Changed with https://github.com/wso2/financial-open-banking/pull/71/commits/b18a4b2a7590752bff54b42ccfd44ae638474986